tests/extra-objects_test.yaml

```diff
@@ -9,9 +9,8 @@ tests:
           count: 0
   - it: should add extra objects
     set:
-      something: world
       extraObjects:
-        - name: "hello {{ .Values.something }}"
+        - name: "hello world"
```

Comment:
Why are we removing the templating part of this test?

Reply:
Because `.Values.something` does not validate against the schema.

On a second thought, I should either remove it altogether or replace with something more useful.

Reply:
Ah gotcha, yeah I think it'd make sense to keep testing that templating works if you can come up with something you like. Using the release name would be a good one.